embassy-nrf/src/chips/nrf5340_app.rs

```diff
@@ -262,7 +262,9 @@ embassy_hal_internal::peripherals! {
     PPI_GROUP5,
 
     // GPIO port 0
+    #[cfg(any(not(feature = "_nrf5340"), feature = "lfxo-pins-as-gpio"))]
```

Comment:
`not(feature = "_nrf5340")` does nothing here because the whole file is only included for nrf5340

Reply:
oops, didn't notice that. thanks!